en/Obsidian Publish/Set up Google Analytics.md

```diff
@@ -0,0 +1,24 @@
+[Google Analytics](https://analytics.google.com) is a service that allows you to track traffic on your website. This page explains how to enable Google Analytics for your [[Introduction to Obsidian Publish|Obsidian Publish]] site.
+
+> **Important:** Before you enable Google Analytics, make sure that your local laws and regulations allow you to track your visitors.
+
+## Prerequisites
+
+To use Google Analytics for Obsidian, you need:
+
+- A Universal Analytics (UA) property (not the new Google Analytics 4 property).
```

Comment:
@lishid Since both are supported, I think we can remove this preqrequisite then?

Reply:
That's correct!